scripts/reducers/bucket.js

```diff
@@ -18,7 +18,10 @@ export function bucket(state = INITIAL_STATE, action) {
       return {...state, busy: action.busy};
     }
     case BUCKET_LOAD_SUCCESS: {
-      const {data} = action;
+      let {bid, data} = action;
+      if (typeof data === "undefined") {
```

Comment:
We should actually improve that patch because since we have a 403 we actually do not have it in listBuckets.